agent/host_agent_suite_test.go

```diff
@@ -46,7 +46,7 @@ var (
 
 const (
 	bundleLookupBaseRegistry = "projects.registry.vmware.com/cluster_api_provider_bringyourownhost"
-	BundleLookupTag          = "v0.1.0_alpha.2"
+	BundleLookupTag          = "v1.22.3"
```

Comment:
do we need this variable? We could reuse K8sVersion itself

Reply:
Spec still has both and didn't change it here.

Reply:
ah okay.